docs/src/updated_example.md

```diff
@@ -0,0 +1,111 @@
+# Uprading docs from Documenter.jl to DocumenterVitepress.jl
+
+Assuming that your current documentation is based on Documenter.jl, you can upgrade to DocumenterVitepress.jl by following these steps:
+
+1. Install DocumenterVitepress.jl using the following julia command in your package's environment:
```

Comment:
maybe add in which folder you are suppose to be, namely `$ docs >`.

Reply:
to be specific, this should be installed in your package's `docs` environment!  The way to do that is `Pkg.activate("path/to/package/docs")`.